tests/network/vmi_multus.go

```diff
@@ -1275,6 +1249,27 @@ var _ = Describe("[Serial]SRIOV", func() {
 	})
 })
 
+func findInterfaceHavingMAC(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance, mac string) string {
+	var ifaceName string
+	err := wait.Poll(140*time.Second, 5*time.Second, func() (done bool, err error) {
```

Comment:
Why is this better than eventually? How this will behave if the error occurs? Before we would have a clear indication of what went wrong.

Also, this effectively set the same timeout for all calls. Before we had much lower timeouts.

Reply:
I am against using gomega's DSL in helper functions - even knowing I could use `...WithOffset` to print the correct errors when they occur. The reason is simple: helpers should not have any kind of expectations: the tests should. The helpers should get values, and whatever is calling them knows what said values should be. 

You make a good point on both cases; I had thought about returning `(string, error)` on this function, but in the end did not because it was so convenient to use it one liner. But you're right in the sense that the error is helpful to understand what went wrong.

Would it be OK if I added the timeout as input parameter, and returned also the error ?

Reply:
@xpivarc ^ forgot to tag you ...

Reply:
Done.